fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/type/DateParmBehaviorUtil.java

```diff
@@ -44,23 +44,23 @@ public DateParmBehaviorUtil() {
     public void executeBehavior(StringBuilder whereClauseSegment, QueryParameter queryParm,
             List<Timestamp> bindVariables,
             String tableAlias) {
-        // Start the Clause 
+        // Start the Clause
         // Query: AND ((
         whereClauseSegment.append(AND).append(LEFT_PAREN).append(LEFT_PAREN);
 
-        // Initially we don't want to 
+        // Initially we don't want to
```

Comment:
really now sure what this one was trying to say; might wanna either complete it or delete it since we're here
```suggestion
```

Reply:
Removed.